src/background/in-dapp-notifications.ts

```diff
@@ -0,0 +1,108 @@
+import { getActiveTabOrigin } from 'src/ui/shared/requests/getActiveTabOrigin';
```

Comment:
Let's move this function out of UI folder?

Reply:
✅ turns out I don't need to import this function at all